c2corg_common/attributes.py

```diff
@@ -608,9 +608,9 @@
 ]
 
 quality_types = [
+    'empty',
```

Comment:
"empty" is used to tag documents that contains almost no info and are not really interesting for visitors (mainly outings that people have created for their logbook). That's not the default document status (would rather be "medium").